### PR TITLE
Create keyFinder.js

### PR DESCRIPTION
**Pull request by winsonrich**
Find and retrieve the encryption key automatically 
Note: This is a draft version, please help to modify, Thanks!

### CODE REVIEW

**Comment by ParthS007** [comment]
This pull request **introduces 1 alert** when merging 5a75bcae529f97f994602b2b36ed3802c202f44b into 7b82729d7ebebab177810ec56e471eef15f39c85 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-a56edb1304fad90a87f1fa0d2ac078092546cc17)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ParthS007** [comment]
This pull request **introduces 1 alert** when merging e0fa99b2fd36b131cbfe607ea15634a3fbd47c7a into 7b82729d7ebebab177810ec56e471eef15f39c85 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-69b9372d19157045b3f26157d37ec8d8814da7a2)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: Ciphers/keyFinder.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/******************************************************
+Find and retrieve the encryption key automatically 
+Note: This is a draft version, please help to modify, Thanks!
+******************************************************/
+function keyFinder(){
+ 	var result = "";
```
Comment (ms10398) [inline Ciphers/keyFinder.js]: Please use ES6 and fix the Error pointed by the check

**Comment by ParthS007** [comment]
This pull request **introduces 1 alert** when merging e41ad76298c9c68eb4871acdaa86eaf27e12a498 into 7b82729d7ebebab177810ec56e471eef15f39c85 - [view on LGTM.com](https://lgtm.com/projects/g/TheAlgorithms/Javascript/rev/pr-614226a102525c2c227db6d3846b9aa3c6578ce7)

**new alerts:**

* 1 for Syntax error

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ms10398** [comment]
I would like you to see this article for usage of let and const instead of var

https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75

**Pull request merged**